### PR TITLE
[DE] Add `Rules/Contributor Code of Conduct`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Translation notes

- `community leader` --> `Führungspersönlichkeit` or `Community-Leiter`
- Not sure how the last two external links should be written (i.e. `Contributor Covenant` and `Mozillas code of conduct enforcement ladder`)

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Rules/Contributor_Code_of_Conduct/de.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+- [peppy](https://osu.ppy.sh/users/2) ([pe@ppy.sh](mailto:pe@ppy.sh))
+- [Ephemeral](https://osu.ppy.sh/users/102335) ([ephemeral@ppy.sh](mailto:ephemeral@ppy.sh))
+
+Alle Beschwerden werden unverzüglich und fair überprüft sowie untersucht.
+
+Alle Community-Leiter sind dazu verpflichtet, die Privatsphäre und Sicherheit des Berichterstatters jedes Vorfalls zu respektieren und zu wahren.
+
+## Richtlinien für die Durchsetzung
+
+Führungspersönlichkeiten der Community werden sich an diese Richtlinien für die Auswirkungen auf die Community richten, um die Konsequenzen jedes Handelns zu bestimmen, das sie als Verstoß gegen diesen Verhaltenskodex ansehen:
```
Comment (The-Last-Cookie) [inline wiki/Rules/Contributor_Code_of_Conduct/de.md:52]:
```suggestion
Die Führungspersönlichkeiten der Community werden sich an diese Richtlinien für die Auswirkungen auf die Community richten, um die Konsequenzen jedes Handelns zu bestimmen, das sie als Verstoß gegen diesen Verhaltenskodex ansehen:
```

File: wiki/Rules/Contributor_Code_of_Conduct/de.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+**Konsequenz**: Eine vorübergehende Sperre von jeder Art von Interaktion oder öffentlicher Kommunikation mit der Community für einen bestimmten Zeitraum. Keine öffentlichen oder privaten Interaktionen mit den involvierten Personen ist während dieser Zeitspanne erlaubt, einschließlich unaufgeforderte Interaktionen mit denen, die den Verhaltenskodex durchsetzen. Ein Verstoß gegen diese Bedingungen kann zu einer permanenten Sperre führen.
+
+### 4. Dauerhafte Sperre
+
+**Auswirkungen auf die Community**: Das Vormachen von Verstößen gegen die Community-Standards, einschließlich anhaltend unangemessenes Verhalten, Belästigung von Einzelpersonen oder Aggression oder Herabwürdigung gegen Personengruppen.
+
+**Konsequenz**: Eine dauerhafte Sperre von jeder Art von öffentlicher Interaktion innerhalb der Community.
+
+## Zuschreibung
```
Comment (The-Last-Cookie) [inline wiki/Rules/Contributor_Code_of_Conduct/de.md:78]: Alternatives:
- Nennung
- Erwähnung
- Danksagung
- Quellen
- `osu!stream` article uses just *Credits*

I'm open to other translations.

File: wiki/Rules/Contributor_Code_of_Conduct/de.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
+# Verhaltenskodex für Mitwirkende
+
+## Unser Versprechen
+
+Wir als Mitglieder, Mitwirkende und Führungskräfte versprechen, die Teilnahme an unserer Community für jeden eine Erfahrung ohne Belästigung zu machen, unabhängig des Alters, der Körpergröße, von sichtbaren oder unsichtbaren Behinderungen, der ethnischen Herkunft, der Geschlechtsmerkmale, der Geschlechtsidentität und des Ausdrucks, des Erfahrungsniveaus, des Bildungsstandes, des sozioökonomischen Status, der Nationalität, des persönliche Aussehens, der Rasse, der Religion oder der sexuellen Identität und Orientierung.
```
Comment (The-Last-Cookie) [inline wiki/Rules/Contributor_Code_of_Conduct/de.md:5]:
```suggestion
Wir als Mitglieder, Mitwirkende und Führungskräfte versprechen, die Teilnahme an unserer Community für jeden eine Erfahrung ohne Belästigung zu machen, unabhängig des Alters, der Körpergröße, von sichtbaren oder unsichtbaren Behinderungen, der ethnischen Herkunft, der Geschlechtsmerkmale, der Geschlechtsidentität und des Ausdrucks, des Erfahrungsniveaus, des Bildungsstandes, des sozioökonomischen Status, der Nationalität, des persönlichen Aussehens, der Rasse, der Religion oder der sexuellen Identität und Orientierung.
```

File: wiki/Rules/Contributor_Code_of_Conduct/de.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+- Konstruktives Feedback geben und taktvoll annehmen
+- Verantwortung akzeptieren und sich bei denjenigen entschuldigen, die von unseren Fehlern betroffen sind sowie aus der Erfahrung lernen
+- Konzentration auf das, was nicht nur für uns als Einzelpersonen, sondern für die gesamte Community am besten ist
+
+Beispiele für inakzeptables Verhalten sind:
+
+- die Verwendung von sexualisierter Sprache oder Bildmaterial und sexuelle Aufmerksamkeit oder Annäherungsversuche jeder Art
+- Trollen, beleidigende oder abwertende Kommentare und persönliche oder politische Angriffe
+- öffentliche oder private Belästigung
+- private Informationen von anderen veröffentlichen, wie zum Beispiel eine physische oder E-Mail-Adresse, ohne das explizite Einverständnis der Person
```
Comment (The-Last-Cookie) [inline wiki/Rules/Contributor_Code_of_Conduct/de.md:24]:
```suggestion
- private Informationen von anderen veröffentlichen, wie zum Beispiel eine physische oder eine E-Mail-Adresse, ohne das explizite Einverständnis der Person
```

File: wiki/Rules/Contributor_Code_of_Conduct/de.md (hunk 2 above)
Comment (The-Last-Cookie) [reply wiki/Rules/Contributor_Code_of_Conduct/de.md:78]:
```suggestion
## Quellen
```

File: wiki/Rules/Contributor_Code_of_Conduct/de.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+**Auswirkungen auf die Community**: Das Vormachen von Verstößen gegen die Community-Standards, einschließlich anhaltend unangemessenes Verhalten, Belästigung von Einzelpersonen oder Aggression oder Herabwürdigung gegen Personengruppen.
+
+**Konsequenz**: Eine dauerhafte Sperre von jeder Art von öffentlicher Interaktion innerhalb der Community.
+
+## Zuschreibung
+
+Dieser Verhaltenskodex wurde aus dem [Contributor Covenant](https://www.contributor-covenant.org) adaptiert, verfügbar unter <https://www.contributor-covenant.org/version/2/0/code_of_conduct.html>.
+
+Die Richtlinien für die Auswirkungen auf die Community sind aus [Mozillas code of conduct enforcement ladder](https://github.com/mozilla/diversity) inspiriert.
```
Comment (The-Last-Cookie) [inline wiki/Rules/Contributor_Code_of_Conduct/de.md:82]:
```suggestion
Die Richtlinien für die Auswirkungen auf die Community sind aus [Mozillas Verhaltenskodex (Englisch)](https://github.com/mozilla/diversity) inspiriert.
```

File: wiki/Rules/Contributor_Code_of_Conduct/de.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+**Auswirkungen auf die Community**: Verwendung einer unangemessenen Ausdrucksweise oder eines anderen Verhaltens, das in der Community als unprofessionell oder unerwünscht gilt.
+
+**Konsequenz**: Eine privat adressierte Warnung von den Community-Leitern, die Klarheit über die Art des Verstoßes schafft und eine Erklärung bietet, warum dieses Verhalten unangemessen war. Eine öffentliche Entschuldigung kann verlangt werden.
+
+### 2. Warnung
+
+**Auswirkungen auf die Community**: Ein Verstoß durch einen einzigen Vorfall oder eine Reihe an Aktionen.
+
+**Konsequenz**: Eine Warnung mit Konsequenzen für anhaltendes Verhalten. Keine Interaktion für einen bestimmten Zeitraum mit den involvierten Personen, einschließlich unaufgeforderte Interaktionen mit denen, die den Verhaltenskodex durchsetzen. Das beinhaltet die Vermeidung von Interaktionen an Community-Orten sowie in externen Kanälen wie Social-Media. Ein Verstoß gegen diese Bedingungen kann zu einer vorübergehenden oder dauerhaften Sperre führen.
```
Comment (FamousPig) [inline wiki/Rules/Contributor_Code_of_Conduct/de.md:64]:
```suggestion
**Konsequenz**: Eine Warnung mit Konsequenzen für anhaltendes Verhalten. Keine Interaktion für einen bestimmten Zeitraum mit den involvierten Personen, einschließlich unaufgeforderter Interaktionen mit denen, die den Verhaltenskodex durchsetzen. Das beinhaltet die Vermeidung von Interaktionen an Community-Orten sowie in externen Kanälen wie Social-Media. Ein Verstoß gegen diese Bedingungen kann zu einer vorübergehenden oder dauerhaften Sperre führen.
```

File: wiki/Rules/Contributor_Code_of_Conduct/de.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+### 3. Temporäre Sperre
+
+**Auswirkungen auf die Community**: Ein schwerwiegender Verstoß gegen die Community-Standards, einschließlich anhaltend unangemessenes Verhalten.
+
+**Konsequenz**: Eine vorübergehende Sperre von jeder Art von Interaktion oder öffentlicher Kommunikation mit der Community für einen bestimmten Zeitraum. Keine öffentlichen oder privaten Interaktionen mit den involvierten Personen ist während dieser Zeitspanne erlaubt, einschließlich unaufgeforderte Interaktionen mit denen, die den Verhaltenskodex durchsetzen. Ein Verstoß gegen diese Bedingungen kann zu einer permanenten Sperre führen.
+
+### 4. Dauerhafte Sperre
+
+**Auswirkungen auf die Community**: Das Vormachen von Verstößen gegen die Community-Standards, einschließlich anhaltend unangemessenes Verhalten, Belästigung von Einzelpersonen oder Aggression oder Herabwürdigung gegen Personengruppen.
```
Comment (FamousPig) [inline wiki/Rules/Contributor_Code_of_Conduct/de.md:74]:
```suggestion
**Auswirkungen auf die Community**: Das Vormachen von anhaltenden Verstößen gegen die Community-Standards, einschließlich anhaltend unangemessenem Verhalten, Belästigung von Einzelpersonen oder Aggression oder Herabwürdigung gegen Personengruppen.
```

File: wiki/Rules/Contributor_Code_of_Conduct/de.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+**Konsequenz**: Eine dauerhafte Sperre von jeder Art von öffentlicher Interaktion innerhalb der Community.
+
+## Quellen
+
+Dieser Verhaltenskodex wurde aus dem [Contributor Covenant](https://www.contributor-covenant.org) adaptiert, verfügbar unter <https://www.contributor-covenant.org/version/2/0/code_of_conduct.html>.
```
Comment (FamousPig) [inline wiki/Rules/Contributor_Code_of_Conduct/de.md:80]:
```suggestion
Dieser Verhaltenskodex wurde aus der [Vereinbarung über den Verhaltenskodex für Mitwirkende](https://www.contributor-covenant.org) adaptiert, verfügbar unter <https://www.contributor-covenant.org/de/version/2/0/code_of_conduct>.
```

Might as well link the official translation imo
https://www.contributor-covenant.org/de/version/2/0/code_of_conduct

File: wiki/Rules/Contributor_Code_of_Conduct/de.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+
+Die Richtlinien für die Auswirkungen auf die Community sind aus [Mozillas Verhaltenskodex (Englisch)](https://github.com/mozilla/diversity) inspiriert.
+
+Für Antworten auf häufige Fragen über den Verhaltenskodex, siehe das FAQ unter <https://www.contributor-covenant.org/faq>. Übersetzungen sind erhältlich unter <https://www.contributor-covenant.org/translations>.
```
Comment (FamousPig) [inline wiki/Rules/Contributor_Code_of_Conduct/de.md:84]:
```suggestion
Für Antworten auf häufige Fragen über den Verhaltenskodex, siehe das FAQ unter <https://www.contributor-covenant.org/faq>. Andere Übersetzungen sind erhältlich unter <https://www.contributor-covenant.org/translations>.
```

In line with above suggestion

**Review by FamousPig — CHANGES_REQUESTED** [review]

**Review by FamousPig — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**